docs/docs/administration/backup-and-restore.md

```diff
@@ -33,7 +33,7 @@ docker start immich_postgres    # Start Postgres server
 sleep 10    # Wait for Postgres server to start up
 gunzip < "/path/to/backup/dump.sql.gz" \
 | sed "s/SELECT pg_catalog.set_config('search_path', '', false);/SELECT pg_catalog.set_config('search_path', 'public, pg_catalog', true);/g" \
-| docker exec -i immich_postgres psql --username=postgres    # Restore Backup
+| docker exec -i immich_postgres psql --username=postgres --dbname=immich    # Restore Backup
```

Comment:
@aviv926 it is not necessary to specify a DB name at restore time with pg_dumpall, which has been the default for a while. Likely that user has an old backup created with tehe backup docker container that doesn't have the POSTGRES_CLUSTER set to true.

Reply:
Is there any downside to this? If not, I think it doesn't hurt to have it because it can solve future problems without affecting future restores (if there is no downside).

Reply:
Hm. I'm doing some testing to see. The biggest issue I see is that when doing a database restore, you can't dump the connecting database. This means that instead of DROP DATABASE Postgres being the one command that fails, now the DROP DATABASE immich command will fail. It's not clear to me if that will cause issues - if it's a fresh DB it shouldn't? 

As an aside, if we do end up re adding it, there is another instance on line 92 that would need it.

Reply:
Thanks for checking, as far as I know the database **must** be completely clean when performing a restore.
That's why we use

```
docker compose create # Create Docker containers for Immich apps without running them.
docker start immich_postgres # Start Postgres server
...
```

Restore and then bring up Immich to start writing data to the database.
For these reasons I don't see it as a problem.

Reply:
We do currently advise that. However, there are installations where it is not possible to start a container (Portainer for example). In these instances, adding dbname=immich will make it harder to restore into these systems. I think whatever we specify it will not work for all users. 

Maybe we can add a note to the page that "if you receive an error when restoring, add / remove dbname=immich". I don't have a strong opinion as to which way, but I think we need to provide both options.

Reply:
I think if it's not required, lets leave this change out for now.